Changes look good, however I'm getting linking errors.
```
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/Piece.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/Pawn.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/Knight.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/Rook.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/Bishop.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/King.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/pieces/Queen.cpp.o
duplicate symbol __ZeqRK8PositionS1_ in:
    CMakeFiles/stocklion.dir/main.cpp.o
    CMakeFiles/stocklion.dir/datastructures/Board.cpp.o
ld: 8 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [stocklion] Error 1
make[1]: *** [CMakeFiles/stocklion.dir/all] Error 2
make: *** [all] Error 2
```
Is it just me?

--- a/datastructures/pieces/Piece.cpp
+++ b/datastructures/pieces/Piece.cpp
@@ -1,5 +1,9 @@
 #include "Piece.h"
 
+bool Position::operator==(const Position &a) {
+    return column == a.column && row == a.row;
+}
+
 void Piece::move(Position p) {
     currentPosition = p;
 }

--- a/datastructures/pieces/Piece.h
+++ b/datastructures/pieces/Piece.h
@@ -11,11 +11,9 @@ enum Color {
 struct Position {
     char column;
     int row;
-};
 
-bool operator==(const Position &a, const Position &b) {
-    return a.column == b.column && a.row == b.row;
-}
+    bool operator==(const Position &other);
+};
 
 
 class Piece {

